OpenRA.Mods.Common/Traits/AttackMove.cs

```diff
@@ -68,6 +68,9 @@ public void ResolveOrder(Actor self, Order order)
 		{
 			if (order.OrderString == "AttackMove" || order.OrderString == "AssaultMove")
 			{
+				if (!self.AcceptsOrder("AttackMove") || !self.AcceptsOrder("AssaultMove"))
```

Comment:
Won't this need to be
`if ((order.OrderString == "AttackMove" && !self.AcceptsOrder("AttackMove")) || (order.OrderString == "AssaultMove" && !self.AcceptsOrder("AssaultMove")))` to work properly? This can then simplify to `if (!self.AcceptsOrder(order.OrderString))`

Reply:
Ah, yes that is way smarter. Updated.